CHANGELOG-3.8.md

```diff
@@ -783,6 +783,45 @@ initialization. See `InitializationData.pluginFactories`.
 - The default value for struct, sequence, and dictionary Slice data members has been changed from `None`
   to a default initialized instance.
 
+- Removed support for the `python:package` Slice metadata. The new `python:identifier` metadata can be used to map modules to
```

Comment:
Should be <= 120 columns.

Reply:
Maybe: Use instead the `python:identifier` metadata directive to map  Slice module to a custom Python package.